src/utils/Utils.re

```diff
@@ -63,138 +63,138 @@ let makeUrl = (network: string, entity: string, id: string) => "/" ++ network ++
 let getFields = (~entity, ~kind=?, ()): array(Type.field) => {
   switch (entity, kind) {
     | ("operation", Some("Transaction")) => [|
-        {name: "operation_group_hash", displayName: "Operation Hash", isLink: false},
-        {name: "kind", displayName: "Kind", isLink: false},
-        {name: "block_hash", displayName: "Block Hash", isLink: true},
-        {name: "timestamp", displayName: "Timestamp", isLink: false},
-        {name: "source", displayName: "From", isLink: true},
-        {name: "destination", displayName: "To", isLink: true},
-        {name: "amount", displayName: "Amount", isLink: false},
-        {name: "fee", displayName: "Fee", isLink: false},
-        {name: "consumed_gas", displayName: "Gas", isLink: false},
-        {name: "parameters", displayName: "Parameters", isLink: false},
-        {name: "status", displayName: "Status", isLink: false}
+        {name: "operation_group_hash", displayName: "Operation Hash", isLink: false, isAddress: false},
+        {name: "kind", displayName: "Kind", isLink: false, isAddress: false},
+        {name: "block_hash", displayName: "Block Hash", isLink: true, isAddress: false},
```

Comment:
isAddress should probably be a type property instead, the way conseil metadata does it